tests/test_subscription.py

```diff
@@ -104,7 +104,7 @@ def test_log_when_execution_is_succesful(
 
     def test_log_does_not_ack_called_message_when_execution_fails(
             self, caplog, message_wrapper):
-        @sub(topic='some-cool-topic')
+        @sub(topic='some-cool-topic', prefix='rele')
```

Comment:
I would suggest adding a test that does not have the prefix `rele` since that is the default if no custom prefix is set.

Reply:
8e2acc5